wavefront/server/apps/floconsole/floconsole/services/floware_proxy_service.py

```
@@ -56,15 +60,31 @@ async def proxy_request(
+        except (ValueError, AttributeError) as e:
+            raise ValueError(
+                f'Invalid user_id format in session: {session.user_id}'
+            ) from e
+
+        # Step 2: Check if user has access to app (RBAC)
+        has_access = await self.user_service.check_user_has_app_access(
+            user_id=user_id, app_id=UUID(app_id)
+        )
+
+        if not has_access:
+            raise ValueError('User does not have access to this app')
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)